Makefile

```diff
@@ -124,6 +124,14 @@ package-lock.json: package.json
 	npm install
 	touch package-lock.json
 
+css: workdir/osm.min.css
+
+workdir/osm.min.css: static/osm.css package-lock.json
+	./node_modules/.bin/cleancss -o $@ $<
```

Comment:
Ezt lehetne valahogy feltételessé tenni? vasony instance-én nincs nodejs, és eddig a nodejs opcionális volt (egyszerűen letöltötte tőlem a bundle.js-t), ezt nem kéne tönkretteni. Egy egyszerű cp megteszi ha a cleancss binary nincs ott, gondolom.

Reply:
Mennyiben más az, mintha az `osm.min.css`-t is átmásolnád neki (vagy ha ő maga másolná át az `osm.css`-t `osm.min.css`-re)? Futtat egyáltalán `make`-et? Ha nem, felesleges. Ha igen, pontosan melyik target-eket?

Reply:
Tettem bele egy feltételt, de a `package-lock.json` még így is kell hozzá. Nem tudom pontosan mit futtat, de jó lenne csinálni neki egy külön targetet, mondjuk `run-nonpm`, és az bemásolni ezt is meg a bundle-t is valahonnan leszedné wget-tel.

Reply:
@vasony ezt le tudod írni, hogy ez hogy megy nálad? És akkor egyben tudnám dokumentálni is, hogy mi a támogatott npm-mentes menet. Köszi!

Reply:
ez a process, ha erre gondoltál:
```
cd $BASEDIR || fail cd $BASEDIR
git pull https://github.com/vmiklos/osm-gimmisn | tee ../github.log || fail git pull

$BASEDIR/../config.sh

wget https://osm-gimmisn.vmiklos.hu/osm/static/bundle.js
#echo touch
touch config.ts
touch package-lock.json
#touch $BASEDIR/workdir/bundle.js
sed -e 's/uriPrefix="\/osm"/uriPrefix=""/'  bundle.js >workdir/bundle.js
rm bundle.js*

make || fail make
```

Reply:
Köszi az információt. Így szemre úgy látom, hogy a `Makefile`-ba újonnan felvett tesztelésnek hála ez továbbra is működni fog nálad.

_/OFF: Csak a vicc kedvéért megírtam bashben is a CSS minifikációt pár sorban ami kb. ugyanennyit megtakarít, de az ilyet nem illendő becommitolni._
- https://github.com/bkil/osm-tooling/commit/6ec356fcfc88a0c15ca4ca54e84e84176bcda23e#diff-2cd19261a1e33882a92ddbacb4498e29657169abfad1aaa98f75fc01a1ab74c4R264